src/components/navigation/_navigation.scss

```diff
@@ -125,6 +130,28 @@
         }
     }
 
+    &--sub-neutral & {
+        &__link {
+            color: var(--ons-color-black);
+            &:hover,
+            &:focus {
+                color: var(--ons-color-black);
+            }
+        }
+        &__item {
+            &:hover {
+                border-color: var(--ons-color-black);
+            }
+            &--active {
+                border-color: var(--ons-color-black);
+            }
+        }
+    }
+
+    &--sub-mobile.ons-navigation--sub-neutral {
+        background: var(--ons-color-white);
+    }
```

Comment:
There are several mention of `--ons-color-black`. It can be updated to:
```
&--sub-neutral & {
        &__link,
        &__link:hover,
        &__link:focus {
            color: var(--ons-color-black);
        }
        
        &__item {
            &:hover,
            &--active {
                border-color: var(--ons-color-black);
            }
        }
    }
```

Reply:
Yeah good spot, fixed 👍